.github/workflows/ci.yaml

```diff
@@ -44,16 +43,16 @@ jobs:
           distribution: ${{matrix.distribution}}
       - name: Install lsof
         run: yum -y install lsof procps
-      - name: Test with Maven
-        run: ./mvnw clean verify -V -B "-Dmaven.artifact.threads=64" "-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn" -DskipToolchain "-Djava.version=${{matrix.java}}" "-Dmaven.javadoc.skip"
+      - name: Test with Gradle
+        run: ./gradlew -V clean check
```

Comment:
The clean shouldn't be needed... should be a no-op.

Reply:
Residue of the Maven version -- removing.